docs/elasticsearch/snapshot-restore.md

```diff
@@ -43,7 +43,7 @@ If you need to delete a snapshot, be sure to use the Elasticsearch API rather th
 
 ## Register repository
 
-Before you can take a snapshot, you have to "register" a snapshot repository. A snapshot repository is really just a storage location: a shared file system, Amazon S3, Hadoop Distributed File System (HDFS), Azure Storage, etc.
```

Comment:
I think the quotes are useful here since "register" is such a meaningless word in this context. I also don't have any issue with etc., e.g., and i.e.